src/components/board/board.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "../../styles/styles.css";
```

Comment:
The board.css should be used here instead, and anything global style be imported into the board.css from styles.css. Maybe rename the file styles.css to be more specific to what it is - main.css or main-styles.css, etc. I know it's styles, but what kind of styles are in there.